packages/frontend/core/src/blocksuite/extensions/theme/preview-theme.ts

```
@@ -0,0 +1,73 @@
+  class AffinePagePreviewThemeExtension
+    extends LifeCycleWatcher
+    implements ThemeExtension
+  {
+    static override readonly key = 'affine-page-preview-theme';
+
+    readonly theme: Signal<ColorScheme>;
+
+    readonly disposables: (() => void)[] = [];
+
+    static override setup(di: Container) {
+      super.setup(di);
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)